### DISCUSSION TITLE
file explorer search plugin dont find anything please help

### DISCUSSION THREAD

**Discussion by NolanNolan** [Flow-Launcher/Flow.Launcher#2515] [Q&A]
HI

i installed file explorer plugin and has fs as keyword followed by the search string i look for, and use windows index ( i have not everythoing.exe installed) but the plugin finds nothing

any suggestions ?

thanks

**Comment by jjw24** [comment]
Make sure your Windows Indexer (Windows Search) is configured to index the paths to the things you are looking to find.

**Comment by NolanNolan** [comment]
Thanks :-)

i had to completely remove the application and reinstall - now it works.

Cant say what the culprit was

IS there any way to make the found files/folders appears sorted woth the latest edited or created at the top?

**Reply by jjw24** [reply]
I think this sorting is only available when using Everything as search engine. Check the Explorer settings.

**Comment by NolanNolan** [comment]
Ok thanks

Hilsen Jørn

Fra: Jeremy Wu ***@***.***>
Sendt: 1. februar 2024 12:08
Til: Flow-Launcher/Flow.Launcher ***@***.***>
Cc: Jørn Bue Madsen ***@***.***>; Author ***@***.***>
Emne: Re: [Flow-Launcher/Flow.Launcher] file explorer search plugin dont find anything please help (Discussion #2515)


I think this sorting is only available when using Everything as search engine. Check the Explorer settings.

**Comment by NolanNolan** [comment]
i now use the search everything application but seems i have to use the version 1.5 in order to index content with search everything application. But then Flow Launcher says "service not running" in the explorer plugin under "everything settings"

If i use version 1.4 of search everything content is not indexed in search everything

See below. What is wrong here ? How to solve ?

![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/30736282/b5bad9c8-24f8-45c3-841f-fbb71fa46583)

**Comment by z1nc0r3** [comment]
Please check the **_Everything_** configurations (Tools > Options) and confirm whether the service is set to running.

![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/64279853/3460ec5b-2c87-4a38-b0a7-6acb8c1283e4)

**Reply by NolanNolan** [reply]
thanks

they ar both checked, but please notice that there is no option at the left pane about content indexing - therefore i suspect that content is NOT indexed. For this to work you need version 1.5 of everything search and that seems not to work woth flow launcher, 

whats your thoughs here ?

**Reply by z1nc0r3** [reply]
I have **Everything 1.4.1.1024** installed and I think there are some issues with the alpha release (**Everything 1.5 Alpha**). Can you try downgrading the **Everything** version?

**Reply by jjw24** [reply]
If I remember correctly, content search is only available for 1.5.

To use 1.5, you need to follow the instructions here https://github.com/Flow-Launcher/Flow.Launcher/issues/1716

Otherwise yes 1.4.1 should work fine

**Reply by NolanNolan** [reply]
Hi "Otherwise version 1.4 should work fine" you wrote, But version 1.4 does not index file content, correct?

Therefore only windows index can be used to search in file content, therefore it puzzles me that there is an option to use everything search for file content, in flow launcher, why is that?

**Reply by NolanNolan** [reply]
i dis as explained in https://github.com/Flow-Launcher/Flow.Launcher/issues/1716 but still get the "warning everything service is not running"

any ideas how to solve ?

**Reply by jjw24** [reply]
If you have Everything installed, Flow Launcher will try and find the installation to use but if you are having issues like getting the not installed error or it is installed in a non standard dirctory, you will need to specify its path explicitly in the Everything settings tab.

**Comment by NolanNolan** [comment]
But version 1.4 does not index file content, correct?

Therefore only windows index can be used to search in file content, therefore it puzzles me that there is an option to use everything search for file content, in flow launcher, why is that?

Hilsen Jørn

On 3. februar 2024 18.14.51 Lasith Manujitha ***@***.***> wrote:

I have Everything 1.4.1.1024 installed and I think there are some issues with the alpha release (Everything 1.5 Alpha). Can you try downgrading the Everything version?

**Reply by NolanNolan** [reply]
But version 1.4 does not index file content, correct?

Therefore only windows index can be used to search in file content, therefore it puzzles me that there is an option to use everything search for file content, in flow launcher, why is that?

**Reply by jjw24** [reply]
> But version 1.4 does not index file content, correct?

From what I heard I think Everything 1.4 is very slow/not supported in content search, so best use Windows Index for it. I myself have never tried to use Everything's content search.

**Comment by NolanNolan** [comment]
I did but flow launcher still display the warning about everything not installed

Hilsen Jørn

On 5. februar 2024 00.00.00 Jeremy Wu ***@***.***> wrote:

If you have Everything installed, Flow Launcher will try and find the installation to use but if you are having issues like getting the not installed error or it is installed in a non standard dirctory, you will need to specify its path explicitly in the Everything settings tab.

**Reply by jjw24** [reply]
Another way is to clear that path completely, then restart flow. It should then prompt you whether you want to install Everything or locate it. Choose locate/browse to Everything file.

Btw just FYI, Everything itself will need to be actually running.

**Comment by NolanNolan** [comment]
it would be so great if the explorer plugin search would arrange the files found after last edited or last accessed (this is the way windows search works) . Is that something that you could consider ? 

I tried to create a suggestin about this feature but for some reason could not.

Is that something that you could consider ?

**Reply by jjw24** [reply]
Assuming you are referring to when using windows index search, yes that's a good option to add.

**Comment by NolanNolan** [comment]
That would be so helpful, would you consider this and would it be very cumbersome to do?

Hilsen Jørn

On 6. februar 2024 22.33.04 Jeremy Wu ***@***.***> wrote:

Assuming you are referring to when using windows index search, yes that's a good option to add.

**Reply by jjw24** [reply]
If windows search returns the search results data that include last modified/accessed dates then it would be easy. Due to other commitments, I probably don't have time to add this feature so it will be up to the rest of the team + others to implement.
